### PR TITLE
User sign up

### PR DESCRIPTION
**Pull request by Managram**

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -26,6 +26,31 @@ angular.module('smartAlarm.controllers', [])
 
 })
 
+.controller('DestinationTimeCtrl', function ($scope, ionicTimePicker, $ionicPopup) {
```
Comment (shaneoston72) [inline www/js/controllers.js]: This controller was removed from master on Thursday.  I am not sure why it is still here.  Please remove the entire DestinationTimeCtrl from your branch please.

**Comment by rhiannonruth** [comment]
🚢

**Comment by MisaOgura** [comment]
:shipit:

**Comment by shaneoston72** [comment]
:shipit:

**Pull request merged**